### ISSUE TITLE
ignore no-duplicate-landmark-elements lint rule for elements displayed conditionally 

### ISSUE THREAD

**Issue by ppara1** [ember-template-lint/ember-template-lint#2534]
Given an hbs file with two forms but only one exists at any given moment, ember-template-lint reports no-duplicate-landmark-elements. Is this intentional or a bug?

```
{{#if this.isCreateProjectFromSavedSearchEnabled}}
   <form></form>
{{else}}
   <form></form>
{{/if}}
```

The form elements here are version1 and version 2 of same feature and cannot have different aria-labels.

**Comment by bmish** [comment]
Some of these rules don't take into account multiple ifs/branches.

**Comment by tylerbecks** [comment]
> Some of these rules don't take into account ifs/branches.

It would be great if they did :) I know it can get complicated with multiple-nested conditionals, but it would be great if this rule could account for a single-layer of if/else

**Comment by bmish** [comment]
Agreed, PRs welcome. Here's an example of one rule that was fixed to handle this:
* #1606
